pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ClientConfigurationData.java

```diff
@@ -62,6 +62,7 @@
     private int connectionTimeoutMs = 10000;
     private long defaultBackoffIntervalNanos = TimeUnit.MILLISECONDS.toNanos(100);
     private long maxBackoffIntervalNanos = TimeUnit.SECONDS.toNanos(30);
+    private int defaultMaxFrameSize = 5242880;
```

Comment:
Actually, this is not needed anymore. The client will not need to configure this anyway

Reply:
But it need initial value to init `defaultFrameDecoder` at first time and it will replace `defaultFrameDecoder` when receive command `Connected` immediately.

Reply:
Oh I can use the value in `Commands`